python/ray/_raylet.pyx

```diff
@@ -635,16 +637,44 @@ cdef execute_task(
 
             # Generate the actor repr from the actor class.
             actor_repr = repr(actor) if actor else None
+            args_info = {}
+
+            if args_resolved:
+                args_name = function_descriptor.args_name.split(",")
+                if (<int>task_type == <int>TASK_TYPE_ACTOR_CREATION_TASK
+                        or <int>task_type == <int>TASK_TYPE_ACTOR_TASK):
+                    # The first element of args_name of
+                    # actor creation / actor tasks is "self", so we pop it.
+                    args_name.pop(0)
+                assert len(args_name) <= len(args)
+                last_named_arg_index = 0
+                # Go through args with the name.
+                # f(a, <- this one b, c)
+                for i, arg_name in enumerate(args_name):
+                    arg = args[i]
+                    args_info[arg_name] = repr(arg)
+                    last_named_arg_index = i
+                # Go through args without the name. e.g., *args
+                # f(a, *args <- this one)
+                for i in range(last_named_arg_index + 1, len(args)):
+                    arg = args[i]
+                    args_info[i] = arg
+                # Go through kwargs.
+                # f(b=3, *, c) <- both of kwargs.
+                for key, arg in kwargs.items():
+                    args_info[key] = repr(arg)
```

Comment:
Can we put this entire thing in a try catch in case we fail to generate the repr?